common/src/main/java/com/viaversion/viaversion/protocols/protocol1_9to1_8/providers/CompressionProvider.java

```diff
@@ -22,7 +22,7 @@ public void handlePlayCompression(UserConnection user, int threshold) {
 
         ChannelPipeline pipe = user.getChannel().pipeline();
 
-        if (threshold == -1) {
+        if (threshold <= 0) {
```

Comment:
0 is a valid threshold (means all packets are compressed)

Reply:
wiki vg says 0 is uncompressed

Reply:
Minecraft Client code: 0 seems valid
![grafik](https://user-images.githubusercontent.com/50594595/130136852-c34644a5-f156-4367-830c-da9f60f9f9e5.png)